pkgs/development/haskell-modules/make-package-set.nix

```diff
@@ -131,7 +131,6 @@ let
       nativeBuildInputs = [ buildPackages.cabal2nix-unwrapped ];
       preferLocalBuild = true;
       allowSubstitutes = false;
-      phases = ["installPhase"];
```

Comment:
Removing this makes little sense, since this is just a one shot action writing a text file.

Should rather be refatcored to use `runCommandLocal`.

Reply:
I also removed  `preferLocalBuild` and `allowSubstitutes`